### DISCUSSION TITLE
Calculate Burn Area

### DISCUSSION THREAD

**Discussion by vikhyatgupta19** [microsoft/PlanetaryComputer#225] [Q&A]
Hello All, 

Am very new to Microsoft Planetary Computer and currently trying to understand its capabilities and functionalities. 

I am trying to calculate burn area year wise from 2012 - 2022 using the dataset MODIS Burned Area Monthly dataset present in data catalog of planetary computer. I would like to know what would be a good starting point as am not sure how to go about it.  I was researching on this and came across this link : https://gis.stackexchange.com/questions/408658/calculating-annual-burned-area-using-modis-006-mcd64a1 

However, I would like to know if same is possible through planetary computer.

Additionally, I would like to know if its possible to upload a shapefile so that the dataset can be clipped to the interest area. 

Any insights on this would be appreciated.

**Comment by TomAugspurger** [comment]
> I am trying to calculate burn area year wise from 2012 - 2022 using the dataset MODIS Burned Area Monthly dataset present in data catalog of planetary computer. I would like to know what would be a good starting point as am not sure how to go about it.


I'm not sure of one, but if you come across an example the please do share!

> Additionally, I would like to know if its possible to upload a shapefile so that the dataset can be clipped to the interest area.


Are you using Python? If so, I would use a library like [fiona](https://fiona.readthedocs.io/en/stable/README.html) to read the shapefile. Then you can rioxararys [`.rio.clip`](https://corteva.github.io/rioxarray/html/rioxarray.html#rioxarray.raster_array.RasterArray.clip) method to crop it to the geometry.

**Reply by vikhyatgupta19** [reply]
Thank you @TomAugspurger for sharing your insights on this.
